homeassistant/components/netgear/device_tracker.py

```diff
@@ -13,27 +13,29 @@
 from homeassistant.const import (
     CONF_DEVICES,
     CONF_EXCLUDE,
-    CONF_HOST,
     CONF_PASSWORD,
+    CONF_HOST,
+    CONF_USERNAME,
     CONF_PORT,
     CONF_SSL,
-    CONF_USERNAME,
 )
 import homeassistant.helpers.config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
 
+CONF_LOG_V2 = "force_login_v2"
 CONF_APS = "accesspoints"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
+        vol.Optional(CONF_DEVICES, default=[]): vol.All(cv.ensure_list, [cv.string]),
+        vol.Optional(CONF_EXCLUDE, default=[]): vol.All(cv.ensure_list, [cv.string]),
+        vol.Required(CONF_PASSWORD): cv.string,
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
Sounds good. May take me longer to implement. Shall I update this PR or create new with expanded scope?

Reply:
Either is fine, depends a bit on how long you think it takes to implement. If you think weeks or longer, it might be a good idea to close it for now and re-open when ready.

Reply:
See https://github.com/home-assistant/core/pull/34779 for an open PR that adds config flow to netgear.

Reply:
Is that moving along okay? Should I work independently towards config flow? Wait for #34779 then add V2 login? Merge this ahead of #34779? I'm open to whatever.

Reply:
Aks the other PR author if you can join forces and finish that PR. We won't merge this PR until there is a config flow in place.